indico/modules/events/layout/default.py

```diff
@@ -97,9 +97,9 @@ def get_default_menu_entries():
             visible=_visibility_call_for_abstracts,
         ),
         MenuEntryData(
```

Comment:
Similar to the `_extend_event_menu` function of the `contributions/__init__.py`?